website/docs/data-display/cigarette-chart/cigarette-chart-d3-code.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Cigarette chart
+fileSource: d3-chart
+tabs: Design('cigarette-chart'), A11y('cigarette-chart-a11y'), API('cigarette-chart-api'), Examples('cigarette-chart-d3-code'), Changelog('d3-chart-changelog')
+---
+
+## Basic usage
+
+::: sandbox
+
+<script lang="tsx">
+  export Demo from './examples/basic-usage.tsx';
+</script>
+
+:::
+
```

Comment:
Could also add an example of use without high level `Chart`?

Reply:
I'm not sure that we should. I didn't add export for CigaretteBar, right now we can use it only from high level Chart.
I've opened another PR (https://github.com/semrush/intergalactic/pull/1284) with example of advanced usage, but I'm not sure that we need to publishing it.